app/femr/ui/views/triage/index.scala.html

```diff
@@ -228,7 +228,7 @@ <h2 class="text-center">Check In - Triage</h2>
                         Submit Patient</button>
                     <button type="button" class="fButton fOtherButton newPatientBtn pull-left">Reset Fields</button>
                 </div>
-
+                <div><h3>Submitting patient in history is for <b>new</b> encounters only.</h3></div>
```

Comment:
AJ,

I think this is from work on https://teamfemr.atlassian.net/browse/FEMR-22 and not the password constraint issue.